numba/core/types/function_type.py

```diff
@@ -52,7 +52,7 @@ def get_call_type(self, context, args, kws):
             # first-class function may not use the same argument names
             # that the caller assumes. [numba/issues/5540].
             raise errors.UnsupportedError(
-                f'first-class function call cannot use keyword arguments')
+                'first-class function call cannot use keyword arguments')
```

Comment:
Is flake8 complaining about a `f` string with no format expression?

Reply:
indeed